packages/framework/src/Resources/views/Admin/Content/PriceList/detail.html.twig

```diff
@@ -4,12 +4,14 @@
 {% block h1 %}{{ 'Price list'|trans }} {{ priceList.name }}{% endblock %}
 
 {% block btn %}
-    <div class="wrap-bar__buttons">
-        <a href="{{ protectedUrl('admin_pricelist_export', {id: priceList.id}) }}" class="btn btn--light wrap-bar__btn">
-            {{ icon('download') }}
-            {{ 'Export CSV'|trans }}
-        </a>
-    </div>
+    {% if priceList is defined %}
```

Comment:
maybe it makes more sense to define the `btn` in `edit` template only - you would not need the `if` there. But I can live with the current solution

Reply:
great idea! fixed